pom.xml

```diff
@@ -128,6 +140,12 @@
 			<version>${packageurl.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.spullara.mustache.java</groupId>

```

Comment:
Is this dependency required at runtime or only during code generation? If the latter then change to `<scope>provided</scope>` as well as adding `<optional>true</optional>`, as this dependency should not leak to consumers.